backend/app/api/endpoints/deliveries.py

```
@@ -371,9 +372,28 @@ def access_loop_item_attachment(
     return LoopItemAttachmentAccessResponse(
-        url=loop_item_service.attachment_access_url(db, attachment_id, current_user.id),
-        expires_in_seconds=900,
+        url=f"wegent://attachments/{attachment_id}",
+        expires_in_seconds=0,
+    )
+
+
+@router.get("/loop-item-attachments/{attachment_id}/content")
+def read_loop_item_attachment(
+    attachment_id: str,
+    db: Session = Depends(get_db),
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)